content/en/docs/contribute/new-content/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Contributing new content
+weight: 20
+---
+
```

Comment:
@celestehorgan , Does this file need any capture statements?

Reply:
I've removed them – I added them in because I haven't encountered any other pages that don't include them.